Api/Resource.php

```diff
@@ -93,7 +93,9 @@ public function getCollectionOperations()
     }
 
     /**
-     * {@inheritdoc}
```

Comment:
The method was no longer defined in the interface so the `@inheritdoc` was not working.